CLAUDE.md

```
@@ -0,0 +1,92 @@
+5. **Apply Least Privilege** - Run with minimal required permissions
+
+## Common Tasks
+
+### Before Committing
+1. Run linting: `ruff check src/`
+2. Run type checking: `mypy src/`
+3. Run tests: `PYTHONPATH=src python -m pytest`
+4. Ensure CI will pass
+
+### Code Review
+- Gemini code review is automatically triggered on PR creation
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)